src/rosdiscover/models/joint_state_publisher.py

```diff
@@ -19,7 +21,9 @@ def get_param(name, value=None):
     get_param("publish_default_positions", True)
     get_param("publish_default_velocities", False)
     get_param("publish_default_efforts", False)
-    get_param("use_gui", False)
+
+    if c.ros_distro < ROSDistribution.NOETIC:
```

Comment:
how does ordering work for ROS1 vs. ROS2?

Reply:
This is the definition for `lt` for `ROSDistribution`:

```
    def __lt__(self, other: Any) -> int:
        if not isinstance(other, ROSDistribution):
            m = f"can only compare ROSDistribution objects"
            raise ValueError(m)
        if self.ros != other.ros:
            m = f"can only compare ROSDistributions for same version"
            raise ValueError(m)
        return self.display_name < other.display_name
```

So I think we're ok.